### PR TITLE
Fix notes.

### PR DESCRIPTION
**Pull request by t-gabor**

### CODE REVIEW

**Comment by t-gabor** [comment]
@dotnet-policy-service agree

File: docs/articles/nunit/writing-tests/attributes/fixturelifecycle.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The constructor of `FixtureLifeCycleAttribute` takes a `LifeCycle` attribute to
 
 ## Notes
 
-* When using `LifeCycle.InstancePerTestCase`, the [`OneTimeSetUp`](xref:onetimesetup-attribute) and [`OneTimeTearDown`](xref:onetimeteardown-attribute) methods must be static, and each are only called once. This is required so that the setup or teardown methods do not access instance fields or properties that are reset for every test.
+* When using `LifeCycle.SingleInstance`, the [`OneTimeSetUp`](xref:onetimesetup-attribute) and [`OneTimeTearDown`](xref:onetimeteardown-attribute) methods must be static, and each are only called once. This is required so that the setup or teardown methods do not access instance fields or properties that are reset for every test.
```
Comment (manfred-brands) [inline docs/articles/nunit/writing-tests/attributes/fixturelifecycle.md:26]: To me the original text was correct. Why do you think otherwise?

Comment (t-gabor) [reply docs/articles/nunit/writing-tests/attributes/fixturelifecycle.md:26]: Yes, it is correct indeed, sorry for the noise.

**Comment by t-gabor** [comment]
The original text is correct.

**Pull request closed without merging**